### PR TITLE
extensions: add crun-wasm to enable wasm support in OCP

### PR DESCRIPTION
**Pull request by sohankunkerkar**
This package enables wasm-supported workloads in OCP. 

xref: https://issues.redhat.com/browse/RFE-4177

### CODE REVIEW

File: extensions-c9s.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@ repos:
   - sig-virtualization
 
 extensions:
+  # https://issues.redhat.com/browse/RFE-4177
+  crun-wasm:
```
Comment (haircommander) [inline extensions-c9s.yaml:10]: should be wasm per update to https://github.com/openshift/machine-config-operator/pull/3776/files

Comment (sohankunkerkar) [reply extensions-c9s.yaml]: Fixed

**Review by jmarrero — APPROVED** [review]

**Comment by marmijo** [comment]
After this merged, `s390x` and `ppc64le` builds started failing in 4.14-9.2. It looks like the crun-wasm package is only built for `x86_64` and `aarch64`, so I opened a PR to limit the new package to only those two arches.
https://github.com/openshift/os/pull/1329

**Comment by LorbusChris** [comment]
This also makes SCOS builds failing, as `crun-wasm` isn't available in any of the specified repos. Is it available elsewhere for C9S @sohankunkerkar ?

**Comment by sohankunkerkar** [comment]
> This also makes SCOS builds fail as `crun-wasm` isn't available in any of the specified repos. Is it available elsewhere for C9S @sohankunkerkar ?

Oh, my bad. It's not available for C9S. We have specifically built it for rhel-9 only: https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2561445

**Comment by sohankunkerkar** [comment]
@LorbusChris FYI: https://github.com/openshift/os/pull/1330

**Pull request merged**